force-app/main/default/classes/SOQL.cls

```diff
@@ -248,7 +281,11 @@ public with sharing class SOQL {
      * @return The Account Name of the Contact.
      */
     public static String getAccountNameForContact(String contactId) {
-        return null; // Replace null with the variable you used to store the result
+        List<Contact> foundContactList = [SELECT Id, AccountId FROM Contact WHERE Id = :contactId]; // Get the Contact for the passed-in Id
+        List<Account> foundAccountList = [SELECT Id, Name FROM Account WHERE Id = :foundContactList[0].AccountId]; // Get the Account for the found Contact
```

Comment:
You should be able to combine this into one query. Look at using dot nation to access related parent records